cmd/operator-sdk/cleanup/cmd.go

```diff
@@ -27,6 +28,23 @@ import (
 	"github.com/spf13/pflag"
 )
 
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "cleanup",
+		Short: "Clean up an Operator deployed with the 'run' subcommand",
```

Comment:
maybe 'run' is not a subcommand?

Reply:
I'm not sure what you mean by this.

Reply:
Sorry for unclear suggestion.

```suggestion
		Short: "Clean up an Operator deployed with the 'run' command",
```